.dir-locals.el

```diff
@@ -0,0 +1,12 @@
+;;; Directory Local Variables
```

Comment:
This just makes life easier for me using Emacs as my IDE... I can prune this file if it's too distracting/irrelavent.

Reply:
I’m not sure we should have this file given that it has hard-coded paths to the install directory, but the user might have installed somewhere else. If this file does need to be included, maybe it could be auto-generated from a template when TAU Commander is installed?